plugins/tnt_roulette/plugin_info.json

```
@@ -0,0 +1,20 @@
+{
+	"id": "tnt_roulette",
+	"authors": [
+		{
+			"name": "LBY123165",
+			"link": "https://github.com/LBY123165"
+		},
+		{
+			"name": "daa0v0",
+			"link": "https://github.com/daa0v0"
+		}
+	],
+	"repository": "https://github.com/LBY123165/tnt_roulette",
+	"branch": "main",
```

```suggestion
	"branch": "main",
	"related_path": "TNT",
```
不必在 repo 里放俩 `mcdreforged.plugin.json`，只留 `TNT/` 那个，然后在这加上 `related_path` 就行